test/language/expressions/class/decorator/metadata/metadata-inheritance.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2023 Ron Buckton. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-runtime-semantics-classdefinitionevaluation
+description: >
+    Metadata is only attached when a class or class element is decorated.

```

Comment:
This description seems copied from somewhere else, maybe something like "Metadata inherits from the metadata of the super class"?